_dashboards/discover/index-discover.md

```diff
@@ -1,104 +1,115 @@
 ---
 layout: default
-title: Exploring data
+title: Discover
 nav_order: 20
 has_children: true
 ---
 
-# Exploring data 
+# Discover
+Updated 2.10
+{: .label .label-purple } 
 
-**Discover** in OpenSearch Dashboards helps you extract insights and get value out of data assets across your organization. Discover enables you to:
+**Discover** is the tool for exploring your data in OpenSearch Dashboards. You can use **Discover** to visually represent your data on a dashboard and provide a high-level view of key metrics. 
 
-1. **Explore data**. You can explore, customize, and filter data as well as search data using [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/dql/).
-2. **Analyze data**. You can analyze data, view individual documents, and create tables summarizing data contents.
-3. **Visualize data**. You can display findings from your saved searches in a single dashboard that combines different data visualization types.
+{::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} The legacy **Discover** tool will be retired in OpenSearch 2.11. The updated **Discover** tool provides you with new features and enhancements that make the data exploration experience in OpenSearch Dashboards more intuitive and user-friendly. To experience the updated tool now, install the [latest version of OpenSearch Dashboards](https://opensearch.org/downloads.html) or try out these features using the [OpenSearch Playground](https://playground.opensearch.org/app/home#/).  
+{: .note} 
 
-## Try it: Exploring sample data with Discover
+In this tutorial, you'll learn about using **Discover** to:
 
-This tutorial shows you how to use Discover to analyze and understand a sample dataset. At the end of this tutorial, you should be ready to use Discover with your own data.
+- Add data
+- Explore and visualize data
+- Share the data findings
+- Set alerts
 
-Before starting this tutorial, make sure you've added the **Sample flight data**. See [Quickstart guide for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/) for information about how to get started.
-{: .warning}
+#### Prerequisites
 
-### Setting up data
+The following are prerequisites for using **Discover** effectively:
 
-Watch the following short video or start with the tutorial steps to learn how to set up a sample dataset in Discover.
+- Download the [latest versions of OpenSearch and OpenSearch Dashboards](https://opensearch.org/downloads.html).
```

Comment:
I would remove the requirement for the latest version and just stated that this tutorial requires OS Dashboards version x.x or later.